person.rb

```diff
@@ -0,0 +1,38 @@
+class Person
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
- [ ] It would be better to remove the id as a parameter and automatically generate one inside the initialize that way it will be safer 
```suggestion
 def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)

```